From: David Milburn <dmilburn@redhat.com>
Date: Thu, 18 Nov 2010 20:40:51 -0500
Subject: [sound] ALSA HD Audio for Intel Patsburg DeviceIDs
Message-id: <1290112856-1796-5-git-send-email-dmilburn@redhat.com>
Patchwork-id: 29509
O-Subject: [RHEL5.6 PATCH 4/9] BZ 570868 ALSA HD Audio patch for Intel Patsburg
	DeviceIDs
Bugzilla: 570868
RH-Acked-by: Stanislaw Gruszka <sgruszka@redhat.com>
RH-Acked-by: Ivan Vecera <ivecera@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Michal Schmidt <mschmidt@redhat.com>

commit cea310e8f8702226f982f09386cfd3c5793c5e2f
Author: Seth Heasley <seth.heasley@intel.com>
Date:   Fri Sep 10 16:29:56 2010 -0700

    ALSA: hda_intel: ALSA HD Audio patch for Intel Patsburg DeviceIDs

    This patch adds the Intel Patsburg (PCH) HD Audio Controller DeviceIDs.

BZ 570868

diff --git a/sound/pci/hda/hda_intel.c b/sound/pci/hda/hda_intel.c
index 3f91b9b..f2736e6 100644
--- a/sound/pci/hda/hda_intel.c
+++ b/sound/pci/hda/hda_intel.c
@@ -129,6 +129,7 @@ MODULE_SUPPORTED_DEVICE("{{Intel, ICH6},"
 			 "{Intel, ICH10},"
 			 "{Intel, PCH},"
 			 "{Intel, CPT},"
+			 "{Intel, PBG},"
 			 "{Intel, SCH},"
 			 "{ATI, SB450},"
 			 "{ATI, SB600},"
@@ -2754,6 +2755,8 @@ static DEFINE_PCI_DEVICE_TABLE(azx_ids) = {
 	{ PCI_DEVICE(0x8086, 0x3b57), .driver_data = AZX_DRIVER_ICH },
 	/* CPT */
 	{ PCI_DEVICE(0x8086, 0x1c20), .driver_data = AZX_DRIVER_PCH },
+	/* PBG */
+	{ PCI_DEVICE(0x8086, 0x1d20), .driver_data = AZX_DRIVER_PCH },
 	/* SCH */
 	{ PCI_DEVICE(0x8086, 0x811b), .driver_data = AZX_DRIVER_SCH },
 	/* ATI SB 450/600 */
